resources/puppetlabs/lein-ezbake/template/global/ext/redhat/ezbake.service.erb

```diff
@@ -10,7 +10,12 @@
 #
 [Unit]
 Description=<%= EZBake::Config[:project] %> Service
-After=syslog.target network.target
+
+<% if EZBake::Config[:build_type] == 'foss' -%>
```

Comment:
testing a build with this change now http://jenkins-release.delivery.puppetlabs.net/job/puppetserver-packaging-2017-05-11-10-23-16-5.0.0.master.SNAPSHOT.2017.05.11T1023/